content/docs/04-clusters/06-cluster-management/01.9-monitoring/10-deploy-monitor-stack.md

```diff
@@ -0,0 +1,289 @@
+
+
+14. Review the YAML configuration. Navigate down the file until you find the parameter `adminPassword`. Input the password value for the admin user. The default admin user name is `admin`.
+
+
+15. Confirm your changes by selecting **Confirm & Create**. There are several options you can enable to expand the functionality of the monitoring stack. Review the [Prometheus Operator](/integrations/prometheus-operator) pack documentation to learn more about the available options.
+
+
+16. Click on **Next** to review the cluster profile. Save the cluster profile.
+
+
+17. Navigate to the left **Main Menu** and select **Clusters**.
+
+
+18. Click on **Add New Cluster**. Select **Deploy New Cluster**.
+
+
+21. Pick the infrastructure provider you selected for the cluster profile you created earlier. 
+
+
+22. Assign a name to the host cluster and select the registered account you wish to deploy the host cluster. Click on **Next**.
+
+
+23. Pick the cluster profile you created earlier and complete the remainder of the cluster creation process.
+
+
+24. Once the host cluster is deployed, navigate to the left **Main Menu** and select **Clusters**. Click on your cluster to display the details page and ensure its status is **Running**. 
+
+
+25. Create a CNAME in your domain's DNS record for the exposed load balancer URL. You can find the load balancer URL by reviewing the cluster details page and reviewing the prefix to any of the exposed service URLs. Discuss this step with your system administrator if you need help and guidance.
+
+
+26. Download the Kubernetes config file. Click on the URL that has the name of your cluster followed by a period and the word *kubeconfig*. Example: `dev-monitoring-stack.config`.
+
+
+27. Open a terminal window and set the environment variable `KUBECONFIG` to point to kubeconfig file you downloaded.
+
+  ```shell
+  export KUBECONFIG=~/Downloads/dev-monitoring-stack.config
+  ```
+
+28. Create a htpasswd file for the user `agent` and assign a password. You can choose a different username if you prefer something else. 
+
+  <br />
+
+  ```shell
+  htpasswd -c auth agent
+  New password: [agent_password_here]
+  New password:
+  Re-type new password:
+  Adding password for user agent
+  ```
+
+28. Convert the htpasswd file into a Kubernetes secret.
+
+  <br />
+
+  ```shell
+    kubectl create secret generic basic-auth --from-file=auth --namespace monitoring
+  ```
+
+  ```shell
+    secret "basic-auth" created
+  ```
+
+29. Navigate back to Palette, and review the cluster profile you created for the monitoring stack. From the left **Main Menu** > **Profiles** > select your cluster profile. Click on the Prometheus operator layer to edit the YAML.
+
+
+30. Locate the `prometheus.ingress` section located towards the end of the file, around line 2244. Update the ingress configuration with the values provided below. Replace the `hosts` parameter with your custom domain.
```

Comment:
It's a very valid point Rita. Unfortunately, because of how large this YAML file is and the repeated instance of ingress and Prometheus, Ctrl + F is not that useful. The best thing I can do is help them by pointing to approximately where the config parameter is.